unrar@7.10: download via aria2 failed

```text
> scoop up
unrar: 7.01 -> 7.10
Updating one outdated app:
Updating 'unrar' (7.01 -> 7.10)
Downloading new version
Starting download with aria2 ...
Download: 02/17 14:03:27 [ERROR] CUID#8 - Download aborted. URI=https://raw.githubusercontent.com/ScoopInstaller/Binary/Download: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://raw.githubusercontent.com/ScoopInstaller/Binary/maDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 2a5479|ERR |       0B/s|C:/Users/jeffm/scoop/cache/unrar#7.10#fecd766.exe
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.10/UnRAR_x64.exe#/UnRAR.exe
    referer=https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.10/UnRAR_x64.exe#/
    dir=C:\Users\jeffm\scoop\cache
    out=unrar#7.10#fecd766.exe

ERROR & 'C:\Users\jeffm\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\jeffm\scoop\cache\unrar.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.5 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=21720 --continue --summary-interval=0 --auto-save-interval=1

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=unrar%407.10%3a+download+via+aria2+failed
```

--- a/bucket/unrar.json
+++ b/bucket/unrar.json
@@ -9,11 +9,7 @@
     "architecture": {
         "64bit": {
             "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.10/UnRAR_x64.exe#/UnRAR.exe",
-            "hash": "90c2b2107a22ea8eb3593a155c4c0007b18b1ba552bf65f963c040038da248be"
-        },
-        "32bit": {
-            "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/7.10/UnRAR.exe",
-            "hash": "c9e67b0474d3081827a73f1a7080b735d1d80d2d5f5c4aee508b335ed7e8a05a"
+            "hash": "9db04f7564a40335c3302a927f604288cd6082d8a0eefd01038d9d5cc9808fa4"
         }
     },
     "bin": "UnRAR.exe",
@@ -22,13 +18,11 @@
         "architecture": {
             "64bit": {
                 "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/$version/UnRAR_x64.exe#/UnRAR.exe"
-            },
-            "32bit": {
-                "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/unrar/$version/UnRAR.exe"
             }
         },
         "hash": {
-            "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/checksums.sha256"
+            "url": "https://raw.githubusercontent.com/ScoopInstaller/Binary/master/checksums.sha256",
+            "regex": "$sha256.*?$version/$basename"
         }
     }
 }

